java/src/test/android/app/src/androidTest/java/ai/onnxruntime/example/javavalidater/SimpleTest.java

```diff
@@ -0,0 +1,74 @@
+package ai.onnxruntime.example.javavalidater;
+
+import ai.onnxruntime.*;
+import ai.onnxruntime.OnnxTensor;
+import ai.onnxruntime.OrtSession;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import android.content.res.AssetManager;
+import android.util.Log;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.platform.app.InstrumentationRegistry;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import java.util.*;
+
+
+@RunWith(AndroidJUnit4.class)
+public class SimpleTest {
+
+    @Test
+    public void runSigmoidModelTest() throws OrtException, IOException {
+
+        try (OrtEnvironment env = OrtEnvironment.getEnvironment();
+             OrtSession.SessionOptions opts = new OrtSession.SessionOptions()) {
+
+            opts.setOptimizationLevel(OrtSession.SessionOptions.OptLevel.BASIC_OPT);
+
+            try (OrtSession session = env.createSession(readModel("sigmoid.ort"), opts)) {
+                String inputName = session.getInputNames().iterator().next();
+                float[][][] testdata = new float[3][4][5];
+                for (float[][] array2d : testdata) {
+                    for (float[] array : array2d) {
+                        Arrays.fill(array, (float) 0);
+                    }
+                }
```

Comment:
Do you want to create the input array with different values (not just all 0s, for example 0 to 3*4*5 or even better with some negative input) and calculate the expected result here directly and use it to compare